src/scss/_status-labels.scss

```diff
@@ -9,38 +9,44 @@
 ));
 
 @include oLabelsAddState('experimental', (
-	background-color: oColorsGetPaletteColor('lemon')
+	background-color: oColorsByName('lemon')
 ));
 
 @include oLabelsAddState('deprecated', (
-	background-color: oColorsGetPaletteColor('crimson')
+	background-color: oColorsByName('crimson')
 ));
 
 @include oLabelsAddState('dead', (
-	background-color: oColorsGetPaletteColor('black')
+	background-color: oColorsByName('black')
 ));
 
 @include oLabelsAddState('master', (
-	background-color: oColorsGetPaletteColor('wheat')
+	// master brand "wheat" colour
+	// (not available to internal brands)
+	background-color: #f2dfce
```

Comment:
I guess we should look to use other colours that are within the internal brand colours?

Reply:
I'm not sure what's best to do here and settled for this for now.

I considered adding it to o-colors for internal brands to reference but I have
before seen an abomination of a UI with a wheat/slate internal mix when
we were first working on branding -- thanks to design/project dev miscommunication.
So I'm keen to limit the internal brand palette and make it clear wheat shouldn't be
used generally.

This is maybe a unique usecase as we are using the label to signify the master brand,
so we kinda do want it wheat, although we shouldn't _really_ worry too much if it gets 
out of sync with actual wheat for this usecase.

The other hexes here which were previously generated using master brand palettes
are replicating a Sass-like pink and a JS-like yellow. We could set custom colour
usecases for these that use the actual Sass/JS colours. But I figured 
master-brand-esqe versions are nice and we don't care if they get out of sync
if the alternative is to use a non-palette colour anyhow :)

Reply:
I added the above to the code comments, and I removed the use of sky which I didn't have a justification for :)

Reply:
Excellent explanation!